app/uploaders/logo_uploader.rb

```diff
@@ -0,0 +1,49 @@
+class LogoUploader < BaseUploader
+  MAX_FILE_SIZE = 3.megabytes
+  EXTENSION_ALLOWLIST = %w[svg png jpg jpeg jpe].freeze
+  IMAGE_TYPE_ALLOWLIST = %i[svg png jpg jpeg jpe].freeze
+  CONTENT_TYPE_ALLOWLIST = %w[image/svg+xml image/png image/jpg image/jpeg].freeze
+
+  def store_dir
+    "uploads/logos/"
+  end
+
+  def extension_allowlist
+    EXTENSION_ALLOWLIST
+  end
+
+  def image_type_whitelist
```

Comment:
I think this should be `image_type_allowlist`, right?
```suggestion
  def image_type_allowlist
```

Reply:
I wouldn't be surprised if CarrierWave expects the other name.

Reply:
Unfortunately we don't have control over the method name 😞 This is method name is expected by [CarrierWave::Bombshell](https://github.com/DarthSim/carrierwave-bombshelter). We've kept our constants to use `allowlist`.